app/handlers/moderator.py

```diff
@@ -103,7 +106,12 @@ async def cmd_warn(message: types.Message, chat: Chat, target: User, user: User)
     text = "Пользователь {user} получил официальное предупреждение от модератора".format(
         user=target.mention_link,
     )
-    await message.reply(text)
+    msg = await message.reply(
+        text,
+        reply_markup=kb.get_kb_warn_cancel(user=user, moderator_event=moderator_event)
+    )
+
+    asyncio.create_task(remove_kb(msg, config.time_to_cancel_actions))
```

Comment:
Are you certain that this action will not block the handler? Telegram requires a response within one minute, and if the handler is sleeping for 30 seconds, it could be problematic. On the other hand, I don't think it's too concerning if the garbage collector sometimes collects our task for removing the keyboard.

Reply:
![image](https://user-images.githubusercontent.com/3669045/224387131-60c76e0d-425d-4ee3-a71b-995d1722d412.png)
as i say - it will be block our handler